crypto/evp/pmeth_sign.c

```diff
@@ -0,0 +1,570 @@
+/*
```

Comment:
I disagree with the name of this file...  `pmeth` files were meant to be all about `EVP_PKEY_METHOD`, and this is not that at all.  How about the pattern set by the KEYEXCH functions, and call it `signature.c`?